### PR TITLE
Add more staking metadata to `NodeStakeUpdate` message

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
Add fields to export the following staking Services properties:
```
### Dynamic properties, can be changed via FileUpdate to 0.0.121
staking.fees.nodeRewardPercentage=0                                              
staking.fees.stakingRewardPercentage=100                                         
staking.maxDailyStakeRewardThPerH=17_808                                                                                           
staking.rewardRate=0                                                         
staking.startThreshold=250_000_000_00_000_000
### Static properties, can only change on network restart
staking.periodMins=1440
staking.rewardHistory.numStoredPeriods=365
```

### CODE REVIEW

File: services/node_stake_update.proto [hunk 1]
```diff
@@ -37,10 +37,50 @@ message NodeStakeUpdateTransactionBody {
...
+   * The fraction of the network and service fees (as a whole number in the range [0, 100]) paid to 
+   * the node reward account 0.0.801.
+   */
+  int32 node_reward_fees_percent = 3;
+
   /**
    * Staking info of each node at the beginning of the new staking period
    */
-  repeated NodeStake node_stake = 2;
+  repeated NodeStake node_stake = 4;
```
Comment (xin-hedera) [inline services/node_stake_update.proto:54]: believe we can't have breaking changes this time, so existing fields number should stay the same:

```suggestion
  repeated NodeStake node_stake = 2;
```

File: services/node_stake_update.proto [hunk 2]
```diff
@@ -37,10 +37,50 @@ message NodeStakeUpdateTransactionBody {
...
+
+  /**
+   * The minimum balance of staking reward account 0.0.800 required to active rewards.
+   */
+  int64 start_threshold = 8;
+
+  /**
+   * The total number of tinybars to be distributed as staking rewards each period.
+   */
+  int64 total_rewards_per_period = 9;
```
Comment (xin-hedera) [inline services/node_stake_update.proto:83]: Q: is this just the sum of `node_stake.reward_rate`? Or `rewardRate`, a.k.a, 

`StakingRewardRate - the total tinybars earned as staking reward during each staking period (example: 100_000_000L * 1_000_000_000 / 365 to emit rewards at a total rate of one billion hbars per year)
` in the HIP?

Comment (tinker-michaelj) [reply services/node_stake_update.proto:83]: It's the `StakingRewardRate` from the HIP. 👍

File: services/node_stake_update.proto [hunk 3]
```diff
@@ -37,10 +37,50 @@ message NodeStakeUpdateTransactionBody {
    */
   Timestamp end_of_staking_period = 1;
 
+  /**
+   * The maximum reward rate, in tinybars per whole hbar, that any account can receive in a day.
+   */
+  int64 max_daily_reward_rate = 3;
```
Comment (steven-sheehy) [inline services/node_stake_update.proto:43]: Will there be a HIP update to add this property? I don't think it's there currently.

**Review by steven-sheehy — CHANGES_REQUESTED** [review]
Requests to align naming with HIP defined names for consistency. I also think the services properties should be named to be consistent with the HIP to avoid confusion.

**Comment by tinker-michaelj** [comment]
> Requests to align naming with HIP defined names for consistency. I also think the services properties should be named to be consistent with the HIP to avoid confusion.

Aligned names; will change Services property names in 0.28. 👍

**Comment by tinker-michaelj** [comment]
> missing field number 5 in `NodeStakeUpdateTransactionBody`. should order fields by its number

Fixed and re-ordered!

**Review by xin-hedera — APPROVED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! But, will need to discuss better approach for notifying mirror node on property changes.

File: services/node_stake_update.proto (hunk 3 above)
Comment (tinker-michaelj) [reply services/node_stake_update.proto]: That could be done. 👀

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**